src/components/OptionsSelector/BaseOptionsSelector.js

```diff
@@ -646,12 +646,21 @@ class BaseOptionsSelector extends Component {
                                     {this.props.translate(`referralProgram.${this.props.referralContentType}.buttonText2`)}
                                 </Text>
                             </Text>
-                            <Icon
-                                src={Info}
-                                height={20}
-                                width={20}
-                                fill={this.props.theme.icon}
-                            />
+                            <PressableWithoutFeedback
```

Comment:
It seems the pressable and the icon have same size.


https://github.com/Expensify/App/assets/46707890/f7abc94f-16df-4118-bb6b-1f40a5f253a3

Shouldn't we have some extra size for pressing? For now both the icon size and the pressing place size is 20

@dannymcclain yours thoughts on this?

Reply:
Good call—ideally the pressable area of the `x` icon should be at least a 44px square.

Reply:
I agree with that Danny. But I could have sworn we can also solve for this using React Native's [hitslop](https://reactnative.dev/docs/touchablewithoutfeedback#hitslop), right? This we we can keep the button where it is but just define a greater hit area around around the button.